src/cms/src/helpers/validator.js

```diff
@@ -14,6 +14,42 @@ module.exports = {
       throw new ApplicationError('Please add protectedArea relation.');
     }
   },
+  // saving without a activityType relation is not allowed
+  activityTypeConnectValidator: function (activityType) {
+    if (activityType?.connect?.length === 0) {
+      throw new ApplicationError('Please add activityType relation.');
+    }
+  },
+  // removing a protectedArea relation is not allowed
+  activityTypeDisconnectValidator: function (activityType) {
+    if (activityType?.disconnect?.length > 0 && activityType?.connect?.length === 0) {
+      throw new ApplicationError('Please add activityType relation.');
+    }
+  },
```

Comment:
Should the comment say activityType instead of protectedArea? The other comments align with the code and the error text, but this one's different

Reply:
Good catch!